CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Fixed incorrect transaction event with respect to a transaction status for when a delegator decreased its stake.
 - Added support for GRPC V2 `GetBlockInfo` for retrieving information of the block being queried.
 - Added support for GRPC V2 `GetTokenomicsInfo` for acquiring various tokenomics related information for a block.
+- Added support for GRPC V2 `GetAncestors`
```

Comment:
Please add a bit context as to what the query informs the user about.

Reply:
done